tools/update-conda.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
```

Comment:
I think it would be good to add an explanation at the top that explains what this script does. Or maybe add a `--help` to the argument parser that prints such documentation.

Reply:
I've now added a bit of documentation in the `README` file in the same directory, and a very short comment on the usage in this file. Do you think this is sufficient?